packages/ui-components/src/index.ts

```diff
@@ -0,0 +1,4 @@
+export {default as LoadingAnimation} from './loading-animation/LoadingAnimation'
+export {default as Page401} from './page-401/Page401'
```

Comment:
for this particular case, I would export a component of `PlaceholdersPage` with individual cases inside it.
I would expect to use it like that `<PlaceholdersPage.401 />` or like that `const {401} = PlaceholdersPage <401 />`